src/Pages/Sponsors/Sponsors.module.scss

```diff
@@ -155,7 +155,16 @@
   .brandGrid2 {
     gap: 3rem;
   }
-  img {
+  .brandLogo1,
+  .brandLogo2,
+  .brandLogo3,
+  .brandLogo4,
+  .brandLogo5,
+  .brandLogomini1,
+  .brandLogomini2,
+  .brandLogomini3,
+  .brandLogomini4,
+  .brandLogomini5 {
     max-width: 30vw;
   }
```

Comment:
If all of the class have same style applied then make them a single classname and use it in the html part of your component